cupyx/jit/_interface.py

```diff
@@ -180,3 +180,16 @@ def __init__(self, name):
 blockDim = _Dim3('blockDim')
 blockIdx = _Dim3('blockIdx')
 gridDim = _Dim3('gridDim')
+
+warpsize = _internal_types.Data(
+    '64' if runtime.is_hip else '32', _cuda_types.uint32)
```

Comment:
`warpsize` can be defined as a constant value.
```suggestion
warpsize = numpy.int32(64'if runtime.is_hip else 32)
```

Reply:
@asi1024 Interesting, thanks. But, I found that in this case we cannot overwrite `warpsize`'s `__doc__` attribute:
```python
>>> a = np.uint32(10)
>>> a.__doc__ = "hahaha"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'numpy.uint32' object attribute '__doc__' is read-only
```
Moreover, this doesn't work with RTD either (tried in fbf602b):
```python
warpsize = ...
"""my docstring"""
```
Am I missing something?

Reply:
I got the point. Thanks!